mono/mini/debugger-agent-stubs.c

```diff
@@ -91,6 +91,12 @@ stub_debugger_agent_breakpoint_from_context (MonoContext *ctx)
 	g_assert_not_reached ();
 }
 
+static void
+mono_debugger_agent_send_crash (char *json_dump, MonoStackHash *hashes, int pause)
+{
+	g_assert_not_reached ();
+}
```

Comment:
Shouldn't this be empty ?

Reply:
I can make it empty. Some of these stubs are empty and some have the ```g_assert_not_reached ();``` , it wasn't really clear what the style was.

Reply:
Made it empty @vargaz